awx_collection/test/awx/conftest.py

```diff
@@ -187,7 +187,7 @@ def mock_load_params(self):
         try:
             result = json.loads(module_stdout)
         except Exception as e:
-            raise Exception('Module did not write valid JSON, error: {0}, stdout:\n{1}'.format(str(e), module_stdout))
+            module.fail_json(msg='Module did not write valid JSON, error: {0}, stdout:\n{1}'.format(str(e), module_stdout))
```

Comment:
We never initialized something like the `module` variable here, if you did, I think it would look like `module = resource_class()`, but I'm not a fan of that pattern for this context